rasa/core/channels/socketio.py

```diff
@@ -99,11 +99,8 @@ async def send_elements(
     async def send_custom_json(
         self, recipient_id: Text, json_message: Dict[Text, Any], **kwargs: Any
     ) -> None:
-        """Sends custom json to the output"""
-
-        json_message.setdefault("room", recipient_id)
-
-        await self.sio.emit(self.bot_message_evt, **json_message)
+        """Sends custom json to the output."""
+        await self.sio.emit(self.bot_message_evt, json_message, room=recipient_id)
```

Comment:
and finally: to finish the PR, we also need to find a way how to test this... do you have any ideas, since you created it?

Reply:
Hi @kalkbrennerei,

I couldn't find any previous `test_socketio.py` file covering unit tests for this channel. I'm not sure how do you want to implement it. If you can help me with some directions I can give it a try

Reply:
Hi @santteegt! You're right, we only have one basic test for this channel, not ideal 😅 It's in `tests/core/test_channels.py`. I think for now a test for this can also be added to the same file. Maybe @degiz has ideas on how to test this?